test/unit/serializer.cpp

```diff
@@ -107,6 +117,40 @@ struct serializer_test
         return t;
     }
 
+    static
+    void
+    check_chunked_body(
+        core::string_view body,
+        core::string_view expected)
+    {
+        for(;;)
+        {
+            auto n = body.find_first_of("\r\n");
+            BOOST_TEST_NE(n, core::string_view::npos);
+            std::string tmp = body.substr(0, n);
+            body.remove_prefix(n + 2);
+            auto chunk_size = std::stoul(tmp, nullptr, 16);
+
+            if( chunk_size == 0 ) // last chunk
+            {
+                BOOST_TEST_EQ(body, "\r\n");
+                body.remove_prefix(2);
+                break;
+            }
+
+            BOOST_TEST_GE(expected.size(), chunk_size);
```

Comment:
The `GE` makes me nervous here. Why can the `expected.size()` exceed the `chunk_size` here? I'd feel more comfortable and secure here is this was `EQ`. `GE` is better for testing properties we can't 100% reliably test. But in this case, a chunk's size should exactly match the chund-header.